ryu/d2s_intrinsics.h

```diff
@@ -189,15 +189,14 @@ static inline uint32_t mod1e9(const uint64_t x) {
 #endif // defined(RYU_32_BIT_PLATFORM)
 
 static inline uint32_t pow5Factor(uint64_t value) {
+  const uint64_t m_inv_5 = 14757395258967641293u; // 5 * m_inv_5 = 1 (mod 2^64)
+  const uint64_t n_div_5 = 3689348814741910323u;  // #{ n | n = 0 (mod 2^64) } = 2^54 / 5
```

Comment:
This comment here is confusing. The value given is [2^64/5], not [2^54/5].

Reply:
Sure. Will fix.

Reply:
Fixed.